python/cuspatial/cuspatial/core/binpreds/binpreds.py

```diff
@@ -85,43 +87,12 @@ def __init__(self, op, lhs, rhs, align=True):
         - `.crosses`
         - `.overlaps`
 
-        There are twenty-five ordering combinations of `lhs` and `rhs`:
-        - point     point
-        - point     multipoint
-        - point     linestring
-        - point     multilinestring
-        - point     polygon
-        - point     multipolygon
-        - multipoint point
-        - multipoint multipoint
-        - multipoint linestring
-        - multipoint multilinestring
-        - multipoint polygon
-        - multipoint multipolygon
-        - linestring point
-        - linestring multipoint
-        - linestring linestring
-        - linestring multilinestring
-        - linestring polygon
-        - linestring multipolygon
-        - multilinestring point
-        - multilinestring multipoint
-        - multilinestring linestring
-        - multilinestring multilinestring
-        - multilinestring polygon
-        - multilinestring multipolygon
-        - polygon point
-        - polygon multipoint
-        - polygon linestring
-        - polygon multilinestring
-        - polygon polygon
-        - polygon multipolygon
-        - multipolygon point
-        - multipolygon multipoint
-        - multipolygon linestring
-        - multipolygon multilinestring
-        - multipolygon polygon
-        - multipolygon multipolygon
+        There are thirty-six ordering combinations of `lhs` and `rhs`, the
+        unordered pairs of each `point`, `multipoint`, `linestring`,
+        `multilinestring`, `polygon`, and `multipolygon`. The ordering of
+        `lhs` and `rhs` is important because the result of the binary
+        operation is not symmetric. For example, `A.contains(B)` is not
+        necessarily the same as `B.within(A)`.
```

Comment:
Why not? Shapely says yes: https://shapely.readthedocs.io/en/stable/manual.html#object.contains

Reply:
Yeah I'll fix this up.